<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>section_settings</class>
 <widget class="QWidget" name="section_settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../gex.qrc">
    <normaloff>:/gex/icons/gex_application_48x48.png</normaloff>:/gex/icons/gex_application_48x48.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,0,0,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBoxTest">
     <property name="title">
      <string>Tests options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="1" column="0" colspan="6">
       <widget class="QFrame" name="frameTestList">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="labelTestFilter">
           <property name="text">
            <string>Filters</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QRadioButton" name="radioButtonNone">
           <property name="text">
            <string>no filter</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QRadioButton" name="radioButtonTopN">
           <property name="text">
            <string>top N failed</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QRadioButton" name="radioButtonTestList">
           <property name="text">
            <string>test list</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QPushButton" name="pushButtonSelectTests">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../gex.qrc">
             <normaloff>:/gex/icons/combo.png</normaloff>:/gex/icons/combo.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="spinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0" colspan="6">
       <widget class="QListWidget" name="listWidgetTests"/>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QTextEdit" name="textEdit">
     <property name="placeholderText">
      <string>Comments...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QFrame" name="frameName">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditName"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QGroupBox" name="groupBoxOptions">
     <property name="title">
      <string>Groups options</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="2" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QListWidget" name="listWidgetGroups"/>
      </item>
      <item row="0" column="0">
       <widget class="QFrame" name="frameSplit">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="3" column="1">
          <widget class="QPushButton" name="pushButtonSelectGroups">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../gex.qrc">
             <normaloff>:/gex/icons/combo.png</normaloff>:/gex/icons/combo.png</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QRadioButton" name="radioButtonGroupList">
           <property name="text">
            <string>group list</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QRadioButton" name="radioButtonGroupNoFilter">
           <property name="text">
            <string>no filter</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="labelGroupFilters">
           <property name="text">
            <string>Filters</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QCheckBox" name="checkBoxSplitByGroup">
           <property name="text">
            <string>split table by Group</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../gex.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>radioButtonTestList</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButtonSelectTests</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonNone</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButtonSelectTests</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonTopN</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButtonSelectTests</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonTopN</sender>
   <signal>toggled(bool)</signal>
   <receiver>listWidgetTests</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonTestList</sender>
   <signal>toggled(bool)</signal>
   <receiver>listWidgetTests</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonNone</sender>
   <signal>toggled(bool)</signal>
   <receiver>listWidgetTests</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonNone</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonTestList</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonTopN</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonGroupNoFilter</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButtonSelectGroups</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>781</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>910</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonGroupList</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButtonSelectGroups</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>652</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>910</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
